runtime/jcl/common/java_lang_Class.cpp

```diff
@@ -1937,4 +1926,22 @@ Java_java_lang_Class_getNestMembersImpl(JNIEnv *env, jobject recv)
 #endif /* JAVA_SPEC_VERSION >= 11 */
 }
 
+jboolean JNICALL 
+Java_java_lang_Class_isHiddenImpl(JNIEnv *env, jobject recv)
+{
+#if JAVA_SPEC_VERSION >= 15
+	jboolean result = JNI_FALSE;
+	J9VMThread *currentThread = (J9VMThread*)env;
+	J9InternalVMFunctions *vmFuncs = currentThread->javaVM->internalVMFunctions;
+	vmFuncs->internalEnterVMFromJNI(currentThread);
+	J9Class *clazz = J9VM_J9CLASS_FROM_HEAPCLASS(currentThread, J9_JNI_UNWRAP_REFERENCE(recv));
+	result = J9_ARE_ALL_BITS_SET(clazz->romClass->extraModifiers, J9AccClassHiddenClass);
```

Comment:
This should be a macro in j9modifiers_api.h

Reply:
Fixed.